addon/components/au-tabs.hbs

```diff
@@ -0,0 +1,5 @@
+<nav class="au-c-tabs {{if @reversed 'au-c-tabs--reversed'}}" ...attributes>
+  <ul class="au-c-tabs-list">
+    {{yield}}
```

Comment:
Impromptu review, but I would yield the component here as well, similar to the  recent AuToolbar PR. I don't think there is a use-case for using the tabs-item component directly?

Reply:
Ok, will fix.